Lib/concurrent/futures/process.py

```diff
@@ -192,11 +183,14 @@ def _process_chunk(fn, chunk):
 def _sendback_result(result_queue, work_id, result=None, exception=None):
     """Safely send back the given result or exception"""
     try:
-        result_queue.put(_ResultItem(work_id, result=result,
-                                     exception=exception))
+        serialize_res = _ForkingPickler.dumps(
+            _ResultItem(work_id, result=result, exception=exception))
     except BaseException as e:
-        exc = _ExceptionWithTraceback(e, e.__traceback__)
-        result_queue.put(_ResultItem(work_id, exception=exc))
+        serialize_res = _ForkingPickler.dumps(_ResultItem(
+            work_id, exception=_ExceptionWithTraceback(e, e.__traceback__)
+        ))
+    result_queue._put_bytes(work_id.to_bytes(WORK_ID_SIZE, WORK_ID_ENC) +
+                            serialize_res)
```

Comment:
This bytes concatenation triggers a potentially very large memory copy which is a regression compared to master.
There are two possible solutions to this problem:

   - Add a send_byte_chunks API down to the Connection that would make it possible to send a topple of bytes without materializing the full buffer. We don't know if it is possible on windows.
   - Implement a new Queue sub class to send and receive 2 messages at a time. This would introduce a small call overhead, potentially critical for small messages.